app/helpers/games_helper.rb

```diff
@@ -1,37 +1,31 @@
 module GamesHelper
   # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity, Metrics/LineLength
   def piece_present?(pieces, game, row_pos, col_pos)
+    white_pieces = {
+      'Pawn'   => '&#9817;',
+      'Rook'   => '&#9814;',
+      'Knight' => '&#9816;',
+      'Bishop' => '&#9815;',
+      'Queen'  => '&#9812;',
+      'King' => '&#9813;'
+    }
+
+    black_pieces = {
+      'Pawn'   => '&#9823;',
+      'Rook'   => '&#9820;',
+      'Knight' => '&#9822;',
+      'Bishop' => '&#9821;',
+      'Queen'  => '&#9818;',
+      'King' => '&#9819;'
+    }
+
     @piece = pieces.find_by(row_position: row_pos, col_position: col_pos)
     if @piece && @piece.user_id == game.white_player_id
-      case @piece.type
-      when "Pawn"
-        white_pawn(@piece, game)
-      when "Rook"
-        white_rook(@piece, game)
-      when "Knight"
-        white_knight(@piece, game)
-      when "Bishop"
-        white_bishop(@piece, game)
-      when "Queen"
-        white_queen(@piece, game)
-      when "King"
-        white_king(@piece, game)
-      end
+      return content_tag(:span, white_pieces[@piece.type].html_safe, class: %w(piece move_mode), id: "#{@piece.id}") if current_user.id == game.white_player_id && game.whos_turn? == game.white_player_id
```

Comment:
You can use a `content_tag` inside another as a method of nesting tags (ie. `<div><p></p></div>`).  However, you're returning a span tag within another span tag.  You could try to just call the `pieces_present?` method on the game show page like so: 

 ```<%= '<td>'.html_safe + piece_present?(@pieces, @game, row, col) + '</td>'.html_safe %>```

You would just have to add the `pos: "#{row}#{col}"` value in the `pieces_present?` method, I think.